lib/foreman/renderer/configuration.rb

```diff
@@ -47,7 +47,8 @@ class Configuration
         :parse_json,
         :to_json,
         :to_yaml,
-        :foreman_server_ca_cert
+        :foreman_server_ca_cert,
+        :time_to_str
```

Comment:
Isn't the name of the new macro `format_time`?..

Reply:
Nice catch! I've fixed it.